### ISSUE TITLE
Problems with CMake: Quick Start

### ISSUE THREAD

**Issue by pauljurczak** [microsoft/vscode-cmake-tools#1004]
I'm using the newest VS Code 1.41.1 and CMake Tools 1.2.3 on Ubuntu 18.04.3. I want to have source files in `hello/src` directory and binaries in `hello/bin` directory. I've set `Cmake: Build Directory` to `${workspaceFolder}/bin` and `Cmake: Source Directory` to `${workspaceFolder}/src`.
When I start VS Code, `File->Open Folder` on empty `hello` directory and run `CMake: Quick Start`, I get:
```
[cms-client] Configuring using the "Unix Makefiles" CMake generator
[rollbar] Unhandled exception: Unhandled Promise rejection: quickStart [cmake-server] Failed to activate protocol version: "sourceDirectory" is not a directory. {}
```
`CMakeLists.txt` and `main.cpp` are created in `hello` directory instead of `hello/src`

I get the same error if I `File->Open Folder` on empty `hello/src`.

When I create an empty `hello/src` and `File->Open Folder` on `hello`, I get `CMakeLists.txt` and `main.cpp` in `hello` directory and `CMakeLists.txt was not found in the root of the workspace folder` pop-up error box.

**Comment by Zingam** [comment]
After you make the settings reload the project and see what happens.

**Comment by pauljurczak** [comment]
I reloaded VS Code right after I made settings.

**Comment by Zingam** [comment]
I tried that on Windows and I got a different result but `Quick Start`  doesn't really work as expected:
![image](https://user-images.githubusercontent.com/3337419/72215223-191ce500-3519-11ea-87f8-2eb1c2165477.png)
1. The generated files were not put in "${workspaceFolder}/src"
2. The configuration step never completed

**Comment by andreeis** [comment]
This should be fixed in CMake Tools 1.5.0 which was published today.
Let us know if you encounter any issues with this release.
